### PR TITLE
Update versions we are shipping

### PR DESCRIPTION
**Pull request by lilic**
Since we did bump these things we should reflect this in both the resources but also this is reflected when users interact via CLI, example:
```
kubectl get prometheus -A
NAMESPACE              NAME   VERSION   REPLICAS   AGE
openshift-monitoring   k8s    v2.15.2   2          154m
```
cc @openshift/openshift-team-monitoring 
* [x] No user facing changes, so no entry in CHANGELOG was needed. (as these were already in the CHANGELOG or jira tickets)

### CODE REVIEW

File: jsonnet/main.jsonnet [hunk 1]
```diff
@@ -80,9 +80,9 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
                  // Because we build OpenShift images separately to upstream,
                  // we have to ensure these versions exist before upgrading.
                  openshiftOauthProxy: 'latest',
-                 prometheus: 'v2.15.2',
-                 alertmanager: 'v0.20.0',
-                 nodeExporter: 'v0.18.1',
+                 prometheus: 'v2.19.2',
+                 alertmanager: 'v0.21.0',
+                 nodeExporter: 'v1.0.1',
                  promLabelProxy: 'v0.1.0',
```
Comment (lilic) [inline jsonnet/main.jsonnet:86]: Not sure if these versions still hold?

Comment (paulfantom) [reply jsonnet/main.jsonnet:86]: no and it never did :D

Comment (lilic) [reply jsonnet/main.jsonnet:86]: did not set that, so didn't update 🤷‍♀️ I meant any other versions other than the ones I set, do we need to update?

Comment (paulfantom) [reply jsonnet/main.jsonnet:86]: This should help our users, so I would say yes we should update them.

Comment (lilic) [reply jsonnet/main.jsonnet:86]: Added to the Release 4.6 checklist #847

**Pull request merged**